cache_test.go

```diff
@@ -413,14 +414,15 @@ func Test_ConcurrentClearAndSet(t *testing.T) {
 		var wg sync.WaitGroup
 
 		cache := New(Configure[string]())
+		wg.Add(1)
```

Comment:
https://pkg.go.dev/sync#WaitGroup.Add:~:text=the%20calls%20to%20Add%20should%20execute%20before%20the%20statement%20creating%20the%20goroutine%20or%20other%20event%20to%20be%20waited%20for.